tkey/src/main/java/com/web3auth/tkey/ThresholdKey/Modules/TSSModule.java

```diff
@@ -0,0 +1,472 @@
+package com.web3auth.tkey.ThresholdKey.Modules;
+
+import android.util.Pair;
+
+import androidx.annotation.Nullable;
+
+import com.web3auth.tkey.RuntimeError;
+import com.web3auth.tkey.ThresholdKey.Common.Result;
+import com.web3auth.tkey.ThresholdKey.Common.ThresholdKeyCallback;
+import com.web3auth.tkey.ThresholdKey.Metadata;
+import com.web3auth.tkey.ThresholdKey.ThresholdKey;
+
+import org.json.JSONObject;
+import org.torusresearch.fetchnodedetails.types.NodeDetails;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.torusresearch.torusutils.TorusUtils;
+import org.torusresearch.torusutils.types.TorusPublicKey;
+import org.torusresearch.torusutils.types.VerifierArgs;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+class GetTSSPubKeyResult {
+    public static class Point {
+        public String x;
+        public String y;
+
+        public String toFullAddr() {
+            return "04" + x + y;
+        }
```

Comment:
Double check this, "04" here is the byte value 4, not 30 32. If this is all a hex string at the end should be fine.